src/client/Client.cc

```diff
@@ -16140,7 +16140,7 @@ int Client::ll_sync_inode(Inode *in, bool syncdataonly)
   if (!mref_reader.is_state_satisfied())
     return -CEPHFS_ENOTCONN;
 
-  ldout(cct, 3) << "ll_sync_inode " << *in << " " << dendl;
+  ldout(cct, 3) << "ll_sync_inode " << in->ino << " " << dendl;
```

Comment:
Why not just put this in the `client_lock` ?

Reply:
Or maybe use `_get_vino(in) ` instead ?

Reply:
> Why not just put this in the `client_lock` ?

Please just ignore this.

Reply:
> Or maybe use `_get_vino(in) ` instead ?

I apologize if I’m not fully familiar with this. Based on the comment in Client::_get_vino, it appears that the caller must hold the client lock, which may not align with our expectations.

Reply:
Thank you for your suggestion. I've submitted a new version accordingly.